CK3toEU4/Data_Files/blankMod/output/common/cultures/!.expanded_culture_groups.txt

```diff
@@ -76,12 +82,11 @@ french = {
 }
 finno_ugric = {
 	komi = {
+		primary = FIN
 		dynasty_names = { Kokko Likhachev Mamin Minin Sibiriak  Tikhanov Zubov }
 	}
 	mordvin = {
-		dynasty_names = { Kokko Likhachev Mamin Minin Sibiriak  Tikhanov Zubov }
-	}
-	khanty = {
```

Comment:
Why are we removing the Khanty?

Reply:
Ostyaki (in EU4) is an older name for the Khanty, they are the same people. The Khanty now convert to the Ostyaki.